launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -65,6 +64,8 @@
       <arg name="remove_unknown" value="$(var remove_unknown)"/>
       <arg name="trust_distance" value="$(var trust_distance)"/>
       <arg name="input/radar" value="$(var input/radar)"/>
+      <arg name="use_low_range_cropbox" value="$(var use_low_range_cropbox)"/>
```

Comment:
How about low_height since low_range would be mistaken for distance?

Reply:
I agree with low_height or low_position. low_range seems misleading

Reply:
@yukkysaito @miursh Thank you for your comments, I changed to low_height here https://github.com/autowarefoundation/autoware.universe/pull/4999/commits/9639f8a10b30b96b30b612dadf4882dcf604bd5b